src/encoded/schemas/meta_workflow_run.json

```diff
@@ -79,6 +79,17 @@
         "cost": {
             "$merge": "encoded_core:schemas/meta_workflow_run.json#/properties/cost"
         },
+        "file_sets": {
```

Comment:
Consider adding a revlink on FileSet so the MWFRs appear there (?). Not sure if desirable.